aspnetcore/security/authentication/customize-identity-model.md

```diff
@@ -39,6 +40,62 @@ When a new app using Identity is created, steps 1 and 2 above have already been
+
+To ensure Identity options are applied consistently during migration generation, use one of the following approaches:
+
+* **Set the startup project:** Run `dotnet ef` commands (or PMC commands) with the application project that calls `AddDefaultIdentity` or `AddIdentityCore` set as the startup project. For example, when running commands from a class library project, specify the startup project with `dotnet ef migrations add {MIGRATION_NAME} --startup-project {PATH_TO_APP_PROJECT}`, where the `{MIGRATION_NAME}` placeholder is the migration name and the `{PATH_TO_APP_PROJECT}` placeholder is the path to the app project.
+* **Implement `IDesignTimeDbContextFactory`:** Alternatively, implement an `IDesignTimeDbContextFactory<TContext>` that constructs the context and applies the equivalent Identity option configuration. For an Aspire-friendly solution, see [efcore#35285 (comment)](https://github.com/dotnet/efcore/issues/35285#issuecomment-3161145762).
+
+Example Identity configuration in `Program.cs`:
+
+```csharp
+builder.Services
+    .AddDefaultIdentity<ApplicationUser>(options =>
+    {
+        options.Stores.SchemaVersion = IdentitySchemaVersions.Version2;
+        options.Stores.MaxLengthForKeys = 256;
+    })
+    .AddEntityFrameworkStores<ApplicationDbContext>();
+```
+
+Example design-time factory:
+
+```csharp
+public class DesignTimeApplicationDbContextFactory
+    : IDesignTimeDbContextFactory<ApplicationDbContext>
+{
+    public ApplicationDbContext CreateDbContext(string[] args)
+    {
+        var optionsBuilder = new DbContextOptionsBuilder<ApplicationDbContext>()
+            .UseSqlServer("{CONNECTION_STRING}");
+
+        return new ApplicationDbContext(optionsBuilder.Options);
+    }
+}
+```
+
+The context itself doesn't automatically know Identity options. If you set model-impacting Identity options such as `options.Stores.MaxLengthForKeys`, you must manually configure the corresponding key lengths in your `ApplicationDbContext.OnModelCreating` method. For example:
+
+```csharp
+protected override void OnModelCreating(ModelBuilder builder)
+{
+    base.OnModelCreating(builder);
+    builder.Entity<IdentityUser>(b =>
+    {
+        b.Property(u => u.Id).HasMaxLength(256);
+    });
+}
+```
+
+> [!NOTE]
+> You cannot access `options.Stores.MaxLengthForKeys` directly inside `OnModelCreating` because dependency injection isn’t available at design time. Instead, specify the configured value directly (such as `HasMaxLength(256)`), or use a design-time mechanism to pass settings if needed.
+> For more details, see <xref:security/authentication/identity-configuration>.
```

Comment:
ASP.NET Identity **does** configure the context based on the options specified, so this section can be removed completely or just leave the Note

Reply:
Thanks @AndriySvyryd for your patience and help.  I completely missed the point of what the framework was now covering automatically and that those details were now obsolete for Identity.

The PR should look better now!